bom/application/pom.xml

```diff
@@ -165,6 +165,7 @@
         <artemis.version>2.11.0</artemis.version>
         <proton-j.version>0.33.6</proton-j.version>
         <okhttp.version>3.14.9</okhttp.version>
+        <okhttp-logging.version>3.14.3</okhttp-logging.version>
```

Comment:
maybe rename it to okhttp3.version?

Reply:
both versions are okhttp3 actually. Its logging specific. thats why I added logging

Reply:
the groupId for okhttp thing is different, isn't it?

Reply:
Is there a reason why we can't use the same version? I see https://repo1.maven.org/maven2/com/squareup/okhttp3/logging-interceptor/3.14.9/ ?

If there is a good reason, let's add a comment. If there is not, let's use the same version and have only one property.

Reply:
@michalszynkiewicz no it's exactly the same. That's why I think there's something fishy with defining 2 inconsistent versions.

Reply:
> the groupId for okhttp thing is different, isn't it?
artifact is different only

Reply:
oh, it's my mistake, I took another look at our analysis file and we indeed use 3.14.9 only for other stuff. @RotBolt could you update this to use okhttp.version for both dependencies?

Reply:
Ok sure updating it